src/menu/columnar_menu.rs

```diff
@@ -667,7 +667,8 @@ impl Menu for ColumnarMenu {
             line_buffer.replace_range(start..end, &value);
 
             let mut offset = line_buffer.insertion_point();
-            offset += value.len().saturating_sub(end.saturating_sub(start));
+            offset += value.len();
+            offset -= end.saturating_sub(start);
```

Comment:
Can we prove that this never underflows? (Not sure if we have enough tests with gnarly insertion points in compleitions in the `reedline` repo for that)

Reply:
good point, I've changed these to saturating_sub/add